package.json

```diff
@@ -9,7 +9,8 @@
     "build-docs": "mkdir -p docs && jsdoc2md src/*.js > docs/api.md",
     "pretest": "npm run lint",
     "test": "jest --testPathIgnorePatterns integration.test",
-    "test:integration": "jest --testPathPattern integration.test"
+    "test:integration": "jest --testPathPattern integration.test",
```

Comment:
`test:integration` suppose to happen only in docker as the image generated in local machine doesn not match the reference one. So I am happy to completely replace this command. So

```
"test:integration": "docker build -t differencify . && docker run --volume \"$(pwd)/src/integration.tests\":/differencify/src/integration.tests -t  differencify npm run test:integration"
```